packages/workers/src/index.ts

```diff
@@ -39,15 +39,19 @@ async function proxyImage(request: Request): Promise<Response> {
 
   const response = await fetch(imageRequest);
   const modifiedResponse = new Response(response.body);
-  modifiedResponse.headers.set('Access-Control-Allow-Origin', '*');
+
+  modifiedResponse.headers.set(
+    'Access-Control-Allow-Origin',
+    request.headers.get('Origin') ?? 'false'
```

Comment:
`'false'` is an invalid value for `Access-Control-Allow-Origin` header, set it to `null` or put it in if the expression: 

```
const origin = request.headers.get('Origin')
if (origin) {
    modifiedResponse.headers.set('Access-Control-Allow-Origin', origin)
}
```

btw the `request.headers.get('Origin')` could be shared between `handler.fetch` and `proxyImage` functions so you don't need to get it from `request.headers` twice

Reply:
See https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Allow-Origin